### ISSUE TITLE
Build entirely from source (without any yum commands)

### ISSUE THREAD

**Issue by fluffynukeit** [RedhawkSDR/redhawk#33]
I want to build and install Redhawk and all supporting libraries entirely from source, meaning without any yum install commands.  Is this feasible?  Is the standalone redhawk EPEL RPM available as a source RPM?  My org is wary of installing any packages from the internet that do not provide us the capability to inspect them prior to compile time.  Thank you.

**Comment by KAUSHAL1986** [comment]
as per the documentations available at "https://redhawksdr.github.io/manual/2.2-lts/", we can install redhawk sdr from source code after compilation. But this procedure works if all the dependencies are installed in linux. But if you want to compile and install those dependencies also it can also be done. But compiling and installing all dependencies will be a big task.

**Comment by fluffynukeit** [comment]
I am fairly familiar with the Redhawk manual and know that RH itself can be build from source, but was not able to find any links in it to *source RPMs* for dependencies instead of binary RPMs.  Obviously Redhawk itself can be built form source.  So is there no standalone Redhawk EPEL source repo?  If Redhawk is packaging its own standalone version of the EPEL, is that standalone EPEL available as a source repo?  I'm presuming the source repo for the standalone version exists somewhere and it is used to create the standalone EPEL binary repo, but it is not linked from the RH manual documentation.  Thanks very much.

**Comment by timcardenuto** [comment]
I'm assuming you're only looking for dependencies that are not in the normal CentOS repo? Those would mostly be:

- libomniEvents2-2.8.2-1.el7.x86_64.rpm
- libomniEvents2-devel-2.8.2-1.el7.x86_64.rpm
- omniEvents-bootscripts-2.8.2-1.el7.x86_64.rpm
- omniEvents-debuginfo-2.8.2-1.el7.x86_64.rpm
- omniEvents-doc-2.8.2-1.el7.x86_64.rpm
- omniEvents-server-2.8.2-1.el7.x86_64.rpm
- omniEvents-utils-2.8.2-1.el7.x86_64.rpm
- python-omniORB-4.2.3-0.52273.el7.noarch.rpm

All of the other stuff is either part of the core Redhawk source or a Redhawk "asset" that can be found on their GitHub. In fact the [omniEvents](https://github.com/RedhawkSDR/omniEvents/releases) dependency is also on their GitHub as a fork, though it appears to be missing the latest tag 2.8.2..... 

So basically it's just the project [omniORB](http://omniorb.sourceforge.net/). I've never built that into RPM's so can't verify that you'll get binary matches on the ones included with Redhawk, but all you need to do is grab the equivalent versions, for Redhawk 2.2.5 it's [omniORB-2.4.3](https://sourceforge.net/projects/omniorb/files/omniORB/omniORB-4.2.3/) and [omniORBpy-2.4.3](https://sourceforge.net/projects/omniorb/files/omniORBpy/omniORBpy-4.2.3/) and build them. There are probably other deps for some of the Redhawk "assets" but that depends.
